nuget/pspdfkit-android.nuspec

```
@@ -4,16 +4,18 @@
         <id>PSPDFKit.Android</id>
         <version>$version$</version>
         <title>PSPDFKit for Xamarin.Android</title>
-        <authors>PSPDFKit GmbH</authors>
-        <owners>PSPDFKit GmbH</owners>
+        <authors>Nutrient</authors>
+        <owners>Nutrient</owners>
         <copyright>Copyright 2010-2021</copyright>
         <license type="file">LICENSE.md</license>
-        <projectUrl>https://pspdfkit.com/</projectUrl>
+        <projectUrl>https://www.nutrient.io/</projectUrl>
         <repository type="git" url="https://github.com/PSPDFKit/Xamarin-Android"/>
         <icon>images\icon.png</icon>
         <requireLicenseAcceptance>false</requireLicenseAcceptance>
         <description>
             The leading framework for displaying, annotating and editing PDFs on iOS, Android, Windows, Electron and the Web.
+
+            The PSPDFKit Xamarin SDK is being sunset and replaced Nutrient .NET for Android (formally PSPDFKit .NET for Android). All current Xamarin licenses are valid and supported through Dec 1, 2025, or until they expire—whichever comes later. The migration from Xamarin to .NET is straightforward, as explained in the official Microsoft guide. Interested in transitioning to another Nutrient product? Contact us and we are happy to help.
```

```suggestion
            Nutrient Xamarin SDK (formerly known as PSPDFKit Xamarin SDK) is being sunset and replaced by [Nutrient .NET for Android](https://www.nutrient.io/guides/android/dotnet/). All Nutrient Xamarin licenses will remain valid and fully supported until 1 December 2025, or until your license expires, whichever occurs later. Interested in transitioning to another Nutrient product? [Contact us](https://www.nutrient.io/company/contact) and we are happy to help.
```

---

@deutschbitte we updated the wording a bit of nuget.org based on [this discussion](https://pspdfkit.slack.com/archives/C1E19K59D/p1730120129426929?thread_ts=1729780903.394409&cid=C1E19K59D) - here is the [reasoning](https://pspdfkit.slack.com/archives/C1E19K59D/p1730116369093089?thread_ts=1729780903.394409&cid=C1E19K59D).

On top of it, I removed mention of Xamarin iOS for Android packages to make the description of package smaller as the package is only related to Android. Similar for iOS package, I removed mention of Android.

Also, I removed links as they are not supported in the description. Considering all this would the following work:
```suggestion
            The PSPDFKit Xamarin SDK is being sunset and replaced by Nutrient .NET for Android. All Xamarin SDK licenses will remain valid and fully supported until 1 December 2025, or until your license expires, whichever occurs later. Interested in transitioning to another Nutrient product? Contact us and we’ll be happy to help.
```

---

Can you add a space between Android and (